src/components/savedAccounts/savedAccounts.css

```diff
@@ -22,9 +22,12 @@
 }
 
 .wrapper {
-  margin: var(--wrapper-margin);
   background: #013165;
   overflow-y: auto;
+  z-index: var(--accounts-index);
+  position: absolute;
+  -webkit-font-smoothing: antialiased;
```

Comment:
Not needed, see https://github.com/LiskHQ/lisk-hub/blob/19a2a613fa08a34fe3088b0e40c11c7fa37a645d/docs/CSS_GUIDE.md#L5

Reply:
not supported by autoprefixer, as is vendor specific, and new spec font-smooth, is not yet supported by all browsers. 

twitter.com/autoprefixer/status/444429500789841921?lang=nl
caniuse.com/#feat=font-smooth